src/graphic/Fast3D/gfx_opengl.cpp

```diff
@@ -847,10 +846,6 @@ static void gfx_opengl_draw_triangles(float buf_vbo[], size_t buf_vbo_len, size_
 }
 
 static void gfx_opengl_init(void) {
-#if !defined(__SWITCH__) && !defined(USE_OPENGLES)
-    glewInit();
-#endif
```

Comment:
why are you fully removing this? just change it to `!defined(__SWITCH__) && !defined(__linux__)`